src/tests/ftest/util/general_utils.py

```diff
@@ -773,24 +773,48 @@ def check_pool_files(log, hosts, uuid):
     return status
 
 
-def convert_list(value, separator=","):
-    """Convert a list into a separator-separated string of its items.
+def list_to_str(value, joiner=","):
+    """Convert a list to a string by joining its items.
+
+    Args:
+        value (list): list to convert to a string
+        joiner (str, optional): _description_. Defaults to ",".
+
+    Returns:
+        str: a string of each list entry joined by the joiner string
+
+    """
+    return joiner.join(map(str, value))
 
-    Examples:
-        convert_list([1,2,3])        -> '1,2,3'
-        convert_list([1,2,3], " ")   -> '1 2 3'
-        convert_list([1,2,3], ", ")  -> '1, 2, 3'
+
+def dict_to_list(value, joiner="="):
+    """Convert a dictionary to a list of joined key and value pairs.
 
     Args:
-        value (list): list to convert into a string
-        separator (str, optional): list item separator. Defaults to ",".
+        value (dict): dictionary to convert into a list
+        joiner (str, optional): string to use to join each key and value. Defaults to "=".
+
+    Returns:
+        list: a list of joined dictionary key and value strings
+
+    """
+    return [list_to_str(items, joiner) for items in value.items()]
+
+
+def dict_to_str(value, joiner=", ", items_joiner="="):
```

Comment:
Seems all uses of `dict_to_str` pass `joiner=','` or `joiner=:`. So does the default of `", "` still make sense?

Reply:
It's almost 50/50 currently. Since everything is passing I'm inclined to leave it the way it is.

Reply:
How can it be 50/50 when there are three options above?

Reply:
What's the resolution here?  Are ", " and "," both valid and considered different?

Reply:
Fine to land with me.